lib/std/os/uefi/pool_allocator.zig

```diff
@@ -43,24 +43,38 @@ const UefiPoolAllocator = struct {
     fn resize(
         _: *anyopaque,
         buf: []u8,
-        log2_old_ptr_align: u8,
+        alignment: mem.Alignment,
         new_len: usize,
         ret_addr: usize,
     ) bool {
         _ = ret_addr;
-        _ = log2_old_ptr_align;
+        _ = alignment;
 
         if (new_len > buf.len) return false;
         return true;
     }
 
+    fn remap(
+        _: *anyopaque,
+        buf: []u8,
+        alignment: mem.Alignment,
+        new_len: usize,
+        ret_addr: usize,
+    ) ?[*]u8 {
+        _ = alignment;
+        _ = ret_addr;
+
+        if (new_len > buf.len) return null;
+        return buf;
```

Comment:
```suggestion
        return buf.ptr;
```

```
std/os/uefi/pool_allocator.zig:68:16: error: expected type '?[*]u8', found '[]u8'
        return buf;
               ^~~
```

Reply:
Woops, that might've been another thing the UEFI PR covers but I forgot to copy over here. I just pushed it in so it should be good now.